src/engine.js

```diff
@@ -147,6 +150,19 @@ Engine.prototype.addLayer = function addLayer (layer) {
 
 /**
  * 
+ * Remove a layer from the engine layersCollection
+ * 
+ * @param {layer} layer - A new layer to be removed from the engine.
+ * 
+ * @public
+ * @api
```

Comment:
I still not see why we're using `@api` here...

Reply:
In my mind we are exposing engine so we need a way to add/remove layers. As you know we have a discussion to decide the final public api 😉 so all this @api tags chill probably be removed